slides/ml_concepts.md

```diff
@@ -23,7 +21,7 @@ coding or maths.
 
 ???
 
-In this first lesson, we will discover what is machine learning and its
+In this video, we will discover what is machine learning and its
```

Comment:
The slides can also be seen outside videos. How about we say "lesson"?
```suggestion
In this lesson, we will discover what is machine learning and its
```

Reply:
True !